unittests/linuxliketests.py

```diff
@@ -668,10 +680,16 @@ def test_install_umask(self):
         '''
         Test that files are installed with correct permissions using default
         install umask of 022, regardless of the umask at time the worktree
-        was checked out or the build was executed.
+        was checked out or the build wasinstall_umask executed.
```

Comment:
How did that get there? :D

Reply:
sigh. My new trackball has a *very* sensitve middle click behavior, so senstive that sometimes running the scroll wheel causes it to paste. Which is the opposite of my old one where the middle click didn't work at all :(